### ISSUE TITLE
Support Android Auto (voice commands) in notifications

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#5128]
We [heard a feature request](https://chat.zulip.org/#narrow/stream/48-mobile/topic/android.20auto.2Fcarplay.3F/near/1282479) to add support for Android Auto. The purpose of this from a user perspective is
> to have the ability to hear the messages read out loud and respond to them with voice commands.

The relevant upstream docs are here:
https://developer.android.com/training/cars/messaging

This will require #4861 as a prerequisite. We'll also need a similar action for "mark as read".

Once we've added those, it looks like this won't require a lot of additional code -- it looks like it involves just adding a bit of metadata to our notifications and those two actions on them. The main work may be in testing it:
https://developer.android.com/training/cars/testing

**Comment by 88fingerslukee** [comment]
Hi @gnprice Do we have any update on the status of this request? It seems that, for Android anyway, the relevant required work should be done?  Thanks for all your efforts!!

**Comment by gnprice** [comment]
No updates; if we had them, they'd be in the thread already.

The original feature request was to be able to reply to Zulip message notifications using voice commands. In order to do that, we'll first need the ability to reply in the first place from a notification context, #4861. There was some progress on that back in 2021, as #4945; but the contributor who built that stopped having time, and it hasn't gotten finished.

@88fingerslukee Is replying to a message the same use case that you have in mind? Or is there another way of using Android Auto that you're more interested in?

**Comment by 88fingerslukee** [comment]
> No updates; if we had them, they'd be in the thread already.
> 
> The original feature request was to be able to reply to Zulip message notifications using voice commands. In order to do that, we'll first need the ability to reply in the first place from a notification context, #4861. There was some progress on that back in 2021, as #4945; but the contributor who built that stopped having time, and it hasn't gotten finished.
> 
> @88fingerslukee Is replying to a message the same use case that you have in mind? Or is there another way of using Android Auto that you're more interested in?
> 

Yes, this is the same use case. I'm looking to reply to messages with my voice while hooked up to android auto
